var/spack/repos/builtin/packages/magma/package.py

```diff
@@ -74,6 +74,9 @@ def cmake_args(self):
         options += ['-DBUILD_SHARED_LIBS=%s' %
                     ('ON' if ('+shared' in spec) else 'OFF')]
 
+        if spec.satisfies('%cce'):
+            options += [('-DCUDA_NVCC_FLAGS=-allow-unsupported-compiler')]
```

Comment:
Not strictly necessary, but eliminates unnecessary parens:

```suggestion
            options += ['-DCUDA_NVCC_FLAGS=-allow-unsupported-compiler']
```

Reply:
Fixed, thanks for reviewing.